buildlib/jucx/jucx-publish.yml

```diff
@@ -1,6 +1,7 @@
 parameters:
   temp_cfg: $(System.DefaultWorkingDirectory)/bindings/java/src/main/native/build-java/tmp-settings.xml
   gpg_dir: $(System.DefaultWorkingDirectory)/bindings/java/src/main/native/build-java/gpg
+  target: publish-snapshot
```

Comment:
why need to set it here?

Reply:
just a default value, to make sure t won't publish releases

Reply:
so maybe it should be 'package'?